projects/plugins/jetpack/extensions/blocks/math/edit.js

```diff
@@ -0,0 +1,103 @@
+import { useBlockProps } from '@wordpress/block-editor';
+import {
+	Button,
+	TextareaControl,
+	/* eslint-disable @wordpress/no-unsafe-wp-apis */
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+	__experimentalText as Text,
+	/* eslint-enable @wordpress/no-unsafe-wp-apis */
+	Placeholder,
+} from '@wordpress/components';
+import { useDispatch } from '@wordpress/data';
+import { useState, useEffect, useRef } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { icon } from './icon';
+import 'katex/dist/katex.min.css';
+import './editor.scss';
+import { renderMath } from './utils';
+
+const LatexPlaceholder = () => {
+	return (
+		<span className="jetpack-math-rendered-placeholder">{ __( 'Write Math…', 'jetpack' ) }</span>
```

Comment:
```suggestion
		<span className="jetpack-math-rendered-placeholder">{ __( 'Write math…', 'jetpack' ) }</span>
```

We try to use sentence case over title case everywhere nowadays.